src/Main.java

```diff
@@ -8,45 +8,46 @@ public static void leapYear(int year){
         }
     }
 
-    public static void phoneVersion (short clientOS, short clientDeviceYear){
+    public static void versionPhone (short clientOS, short clientDeviceYear){
```

Comment:
```suggestion
    public static void printInfoAboutVersion(short clientOS, short clientDeviceYear){
```
сразу понятно по названию, что метод делает